tests/acceptance/features/webUITrashbinRestore/trashbinRestore.feature

```diff
@@ -134,73 +134,73 @@ Feature: Restore deleted files/folders
-    And the following files have been deleted by user "Alice" in the server
-      | name                                     |
-      | simple-folder/file-to-delete-and-restore |
-    And user "Alice" has renamed folder "simple-folder" to "simple-folder-renamed" in the server
-    When the user browses to the trashbin page
-    And the user restores file "simple-folder/file-to-delete-and-restore" from the trashbin using the webUI
-    Then the following error message should be displayed on the webUI
-      """
-      Failed to restore "file-to-delete-and-restore"
-      """
-    And as "Alice" file "simple-folder-renamed/file-to-delete-and-restore" should not exist in the server
-    And as "Alice" file "simple-folder/file-to-delete-and-restore" should not exist in the server
-
-  @issue-1753 @skipOnOCIS @issue-product-186
-  Scenario: Restore a file from trashbin without restoring the parent folder (bug demonstration)
-    Given user "Alice" has created folder "simple-folder" in the server
-    And user "Alice" has created file "simple-folder/file-to-delete-and-restore" in the server
-    And the following files have been deleted by user "Alice" in the server
-      | name                                     |
-      | simple-folder/file-to-delete-and-restore |
-      | simple-folder                            |
-    When the user browses to the trashbin page
-    And the user restores file "simple-folder/file-to-delete-and-restore" from the trashbin using the webUI
-    Then the following error message should be displayed on the webUI
-      """
-      Failed to restore "file-to-delete-and-restore"
-      """
-    And as "Alice" the file with original path "simple-folder/file-to-delete-and-restore" should exist in the trashbin in the server
-    And as "Alice" file "simple-folder/file-to-delete-and-restore" should not exist in the server
-
-  @skipOnOC10 @issue-product-186 @issue-ocis-1057
-  Scenario: Restore a file from trashbin without restoring the parent folder (ocis bug demonstration)
-    Given user "Alice" has created folder "simple-folder" in the server
-    And user "Alice" has created file "simple-folder/file-to-delete-and-restore" in the server
-    And the following files have been deleted by user "Alice" in the server
-      | name                                     |
-      | simple-folder/file-to-delete-and-restore |
-      | simple-folder                            |
-    When the user browses to the trashbin page
-    And the user restores file "simple-folder/file-to-delete-and-restore" from the trashbin using the webUI
-    Then the following error message should be displayed on the webUI
-      """
-      Failed to restore "file-to-delete-and-restore"
-      """
-    And as "Alice" file "simple-folder/file-to-delete-and-restore" should not exist in the server
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Why all these empty lines?

Reply:
You never just delete lines in feature files if you can avoid it. Replacing with empty lines is the way to go, because this way the line numbers of other test scenarios are not modified. Why is this good? Because we have `expected to fail`-files which define expected failures by line numbers.